include/extractor/turn_instructions.hpp

```diff
@@ -9,21 +13,22 @@ namespace extractor
 enum class TurnInstruction : unsigned char
 {
     NoTurn = 0,
-    GoStraight,
-    TurnSlightRight,
-    TurnRight,
-    TurnSharpRight,
-    UTurn,
-    TurnSharpLeft,
-    TurnLeft,
-    TurnSlightLeft,
-    ReachViaLocation,
-    HeadOn,
+    GoStraight,       // 1
+    TurnSlightRight,  // 2
+    TurnRight,        // 3
+    TurnSharpRight,   // 4
+    UTurn,            // 5
+    TurnSharpLeft,    // 6
+    TurnLeft,         // 7
+    TurnSlightLeft,   // 8
+    ReachViaLocation, // 9
+    HeadOn,           // 10
```

Comment:
What are the comments for here? If you want to be explicit, why not saying `GoStrainght = 1,` and so on?

Also what about the `127`, `129` below? Do their numeric values have specific meaning?